setup.py

```diff
@@ -39,6 +39,7 @@
     "pytest",
     "pytest-runner",
     "click",
+    "sqlalchemy-utils",
```

Comment:
note that we're now starting to test integration with a non-SQLAlchemy library. This is one of many other alternative libraries which enhance SQLAlchemy.